shared/actions/favorite.js

```diff
@@ -161,6 +194,30 @@ export function favoriteList (): (dispatch: Dispatch) => void {
   }
 }
 
-export function ignoreFolder (path: string) {
-  return () => console.log('TODO: implement ignore folder action')
+export function ignoreFolder (path: string): (dispatch: Dispatch) => void {
+  return (dispatch, getState) => {
+    const folder = folderFromPath(path)
+    if (!folder) {
+      const action: FavoriteIgnore = {type: Constants.favoriteIgnore, payload: {error: true}}
```

Comment:
Sorry I wasn't clear, I meant adding the error flag up one level, like this: https://github.com/keybase/client/blob/master/shared/actions/signup/index.js#L47

and then for the non error case:

https://github.com/keybase/client/blob/master/shared/actions/signup/index.js#L50